src/templates/actors/fragments/vehicle-fitting-table.html

```diff
@@ -6,7 +6,7 @@
     <div class="item-controls text-right">
     
       <a class="item-control item-search text-gray-100 hover:text-green-400 transition-colors"
-        title="{{localize 'swnr.sheet.search-item'}}" data-item-type="shipFitting"  data-item-name="Fitting">
+        title="{{localize 'swnr.sheet.search-item'}}" data-item-type="shipFitting"  data-item-name="Fitting" data-item-subtype={{data.type}}>
```

Comment:
everything looks good, but i am guessing this would need to be added to other files (vehicle-weapon/def-table).  also might be good to pass the subtype as an argument to the fragment as there is no guarantee that the type matches the field (it actually could but might be cleaner to be specify) see https://github.com/wintersleepAI/foundry-swnr/blob/c46c2548038eb337c4a0fae794114cbe7d000a45/src/templates/actors/drone-sheet.html#L201 

Thoughts?

Reply:
My initial thought is that in the SWN ruleset there **aren't** fittings of the other types for anything but ships. 

That said maybe some 3pp adds non ship wep and def fittings? 

Either way, can't hurt?

Reply:
Well this was in part a bad design by me. Drones, vehicles, mechs, and ships all share the same item type which are shipWeapons, shipFittings, and shipDefenses. The data.type (  type: "ship" | "drone" | "mech" | "vehicle") specifies what is it for, which really only changes the html form for the type.